README.md

```diff
@@ -28,6 +28,21 @@ use({
 })
 ```
 
+You can also supply optional argument to the setup function if you want to
+enable our experimental feature.
+
+```lua
+require("neotest").setup({
+  adapters = {
+    require("neotest-go")({
+      x = { 
```

Comment:
I think just calling this experimental would be better? Since `x` is not really very clear.

Reply:
I thought just `x` was going to be a cool option name. But updated it regardless.